live-examples/html-examples/inline-text-semantics/q.html

```diff
@@ -1,6 +1,6 @@
 <p>
   When Dave asks HAL to open the pod bay door, HAL answers:
-  <q cite="https://www.imdb.com/title/tt0062622/quotes/?item=qt0396921&ref_=ext_shr_lnk">
+  <q cite="https://www.example.com/movies/2001-space-odyssey/quotes/?item=2">
```

Comment:
this removes a little bit of entertainment from MDN. I am not sure I agree with removing actual links to  replace with fake `example.com` links in all cases. Is there a discussion I missed?

Reply:
@estelle https://github.com/orgs/mdn/discussions/450
If live-example causes real traffic to the site, it should be avoided since MDN has a large number of users.
Example domains are standardized for this purpose in [RFC6761](https://datatracker.ietf.org/doc/html/rfc6761).
MDN should respect it.

Reply:
cc @bsmth